src/AvtaleSide/AvtaleOversikt.tsx

```diff
@@ -52,6 +55,17 @@ const AvtaleOversikt: FunctionComponent<RouteComponentProps> = props => {
                         {moment(avtale.opprettetTidspunkt).format('DD.MM.YYYY')}
                     </div>
                 </MediaQuery>
+                <div className="avtaleoversikt__lenker__status">
+                        {avtale.status}
+                </div>
+                <img
```

Comment:
Ref. kommentar over, så kan du bytte med en react-komponent i stedet for `<img src="..." />` så gjør man `<GodkjenningIkon .../>`